docs/pages/getting-started/3-configure-your-node.rst

```diff
@@ -56,8 +56,9 @@ This results in 3 files:
 Bootstrap nodes
 ===============
 
-A bootstrap node is just a normal Nuts node which is available for other nodes to connect to. When you want to join a network, you must approach another network participant and ask for its public endpoint. After connecting, this node will sent an address list of other connected nodes (this requires a Network Identity and NutsComm endpoint, see further down for more information). Your node must connect to the GRPC endpoint which is usually configured on the ``5555`` port.
-Consult the community on `Slack <https://nuts-foundation.slack.com/>`_ to find out which public bootstrap nodes are available to connect to your network of choice.
+A bootstrap node is just a normal Nuts node which is available for other nodes to connect to. When you want to join a network, you must approach another network participant and ask for its public endpoint. After connecting, this node will sent an address list of other connected nodes (this requires you to configure a Network Identity and NutsComm endpoint, see further down for more information). Your node must connect to the others GRPC endpoint which is usually configured on the ``5555`` port.
```

Comment:
Ok, ambiguity is gone but now it's incorrect. You don't need an identity to receive DID Documents and/or connect to other nodes.